common/app/navigation/Navigation.scala

```diff
@@ -146,16 +146,28 @@ object NavMenu {
     s"/$id"
   }
 
-  private[navigation] def getSubnav(currentNavLink: Option[NavLink], currentParent: Option[NavLink], currentPillar: Option[NavLink]): Subnav = {
-    val currentNavHasChildren = currentNavLink.exists(_.children.nonEmpty)
-    val parentIsPillar =  currentParent.equals(currentPillar)
-    def currentNavIsPillar = currentNavLink.equals(currentPillar)
+  private[navigation] def getCustomSignPosting(navItem: NavItem): Subnav = {
+    val children = navItem.links.map(link => {
+      NavLink(link.breadcrumbTitle, link.href)
+    })
+    val parent = NavLink(navItem.name.breadcrumbTitle, navItem.name.href)
+    Subnav(Some(parent), children)
+  }
+
+  private[navigation] def getSubnav(page: Page, currentNavLink: Option[NavLink], currentParent: Option[NavLink], currentPillar: Option[NavLink]): Subnav = {
+    page.metadata.customSignPosting.map( customSignPosting => getCustomSignPosting(customSignPosting)).getOrElse {
```

Comment:
you don't need to name the `customSignPosting` val. You can simply call `page.metadata.customSignPosting.map(getCustomSignPosting)`